pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -156,8 +156,20 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
```

Comment:
@ironcladlou can you take a look at this set of permissions, and see if it is sufficient for the client the deployment pod runs as?